toolchain/driver/compile_subcommand.cpp

```diff
@@ -862,6 +874,10 @@ auto CompileSubcommand::Run(DriverEnv& driver_env) -> DriverResult {
         // TODO: Decide if we want this.
         "-fPIE",
     };
+    if (driver_env.fuzzing && !options_.clang_args.empty()) {
+      DisableFuzzingExternalLibraries(driver_env, "compile");
+      return {.success = false};
+    }
```

Comment:
Maybe a comment for the future readers?
```suggestion
    if (driver_env.fuzzing && !options_.clang_args.empty()) {
      // Parsing specific Clang arguments can reach deep into
      // externel libraries that aren't fuzz clean.
      DisableFuzzingExternalLibraries(driver_env, "compile");
      return {.success = false};
    }
```